### DISCUSSION TITLE
How do I override border-radius of NumberInput stepper?

### DISCUSSION THREAD

**Discussion by Tymek** [chakra-ui/chakra-ui#5763] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/2625371/159480515-15112a70-25bc-41f3-80fc-39f676422341.png)

I'd like to remove border radius on this corner. Without success I tried:

```js
  NumberInput: {
    parts: ['stepper', 'stepperGroup'],
    baseStyle: {
      stepper: {
        borderRadius: 'none',
      },
      stepperGroup: {
        borderRadius: 'none',
      },
    },
  },
```

**Comment by chr-ge** [comment, marked as answer]
The `borderRadius` is handled by the various `sizes` of the `NumberInput`. So you will have to create a custom size that "extends" one of the existing sizes (xs/sm/md/lg).

Here is a [codesandbox example](https://codesandbox.io/s/chakraui-no-border-numberinput-n2vrv4?file=/src/index.tsx).
